code/game/objects/items/clown_items.dm

```diff
@@ -127,20 +128,27 @@
 	. = ..()
 	AddComponent(/datum/component/squeak, list('sound/items/airhorn2.ogg'=1), 50)
 
+//golden bikehorn
 /obj/item/bikehorn/golden
 	name = "golden bike horn"
 	desc = "Golden? Clearly, it's made with bananium! Honk!"
 	icon_state = "gold_horn"
 	item_state = "gold_horn"
+	var/flip_flag = FALSE
 
 /obj/item/bikehorn/golden/attack()
-	flip_mobs()
+	if (flip_flag == FALSE)
+		flip_mobs()
 	return ..()
 
 /obj/item/bikehorn/golden/attack_self(mob/user)
-	flip_mobs()
+	if (flip_flag == FALSE)

```

Comment:
if(!flip_flag)

Reply:
oooh, I didn't think these were automatically defined. Roger that.